torch/csrc/autograd/engine.cpp

```diff
@@ -108,6 +153,8 @@ struct GraphTask {
 
   void init_to_execute(Function& graph_root, const edge_list& captures);
 
+  // This member variable is used to handle reentrant backwards
+  // execution.  See Note [Reentrant backwards]
```

Comment:
That could be made much more helpful by saying "The value of `worker_device` in the thread that created this task. See Note [...]"